MetaMask/metamask-extension
# Fix/19549 decrypt request shows partial message

## Explanation

Fixes #19549: Unable to view the full length of a long decrypted message in the decryption request prompt.

Applied `overflow: auto` to decryption message box to allow scrolling. Adjusted padding to prevent overlap of text and scroll bar.

### Before

https://github.com/MetaMask/metamask-extension/assets/4392888/43194e63-d2b9-449c-8e51-d12ebea1163f

### After

https://github.com/MetaMask/metamask-extension/assets/4392888/f3833beb-1649-42f0-8ffd-bcfbe0a3d76c

## Manual Testing Steps

1. Go to [test dapp](https://metamask.github.io/test-dapp/)
2. Connect wallet
3. Click 'Get Encryption Key' and accept
4. Paste a long text in the input field
5. Click 'Encrypt'
6. Click 'Decrypt'
7. Click 'Decrypt Message' in MM
8. Hover cursor over decryption text
9. Verify text is scrollable and entire message can be viewed

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
